data-dashboard/pages/signin.tsx

```diff
@@ -79,13 +80,24 @@ const ImageStyle = {
   marginRight: '202px',
 };
 
+const ErrorMessage = styled.p`
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  color: #e71d36;
+  margin-top: 8px;
+`;
+
 function SignIn() {
   const router = useRouter();
+  const [error, setError] = useState(false);
   const { mutate } = trpc.auth.login.useMutation({
```

Comment:
Please use trpc error instead:
```suggestion
  const { mutate, error } = trpc.auth.login.useMutation({
```